content/posts/2015-05-03_may-newsletter.md

```diff
@@ -0,0 +1,137 @@
+Title: May Newsletter
+Date: 2015-05-03
+Tags: newsletter
+Slug: may-newsletter-2015
+Author: David Fischer
+
+
+Pythonistas,
+
+We are always looking to highlight upcoming events, happenings and news related to the local Python community. If you come across something that should be in the newsletter, [let us know][].
+
+[let us know]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+Upcoming events
+---------------
+
+
+[Saturday study group][saturday-meetup] (meetup.com) <br />
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at the **Ansir Center**. The goal of the Saturday study group is to learn Python
+together in a collaborative environment.
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/221953664/
+
+
+[PyLadies Casual Study Group][] (meetup.com) <br />
+San Diego PyLadies are meeting at Lestat's on Adams Avenue on Tuesday May 12
+at 6pm. Whether you are interested in learning Python or a Python pro looking
+to keep your skills fresh with a friendly group of Ladies and supportive men,
+this is the group for you. This meeting is now regularly held on the 2nd and
+4th Tuesdays.
+
+[PyLadies Casual Study Group]: http://www.meetup.com/sd-pyladies/events/222165575/
+
+
+[Tijuana Python Users Group][] (facebook.com) <br />
+May 14 will be the first meeting of the Tijuana Python Users Group. The event
+will be in Spanish at the fantastic HUB STN coworking space at 7pm.
+
+[Tijuana Python Users Group]: https://www.facebook.com/events/1620616498151586/
+
+
+[Monthly Meetup][] (meetup.com) <br />
+Our regular monthly meetup will be held at the Ansir Innovation Center on
+Thursday May 28 at 7pm. We are actively seeking speakers for lightning talks.
```

Comment:
Maybe we should specify "5 minute lightning talks"?  The JS group has been doing 10 and 15 minute "lightning talks" while we're been sticking to a strict 5.